app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/AlertPlayer.java

```diff
@@ -237,21 +242,24 @@ private void Vibrate(Context ctx, AlertType alert, String bgValue, Boolean overr
                 if(overrideSilent) {
                     PlayFile(ctx, alert.mp3_file, volumeFrac);
                 } else {
-                    builder.setSound(Uri.parse(audioPath), AudioAttributes.USAGE_ALARM);
+                    builder.setSound(Uri.parse(audioPath));
                 }
             }
```

Comment:
I believe that this is not what you meant to do.
In other words, now it will only vibrate in the case of vibrate only, while we want it to vibrate in all casses that are not silent.

Seems to me that in the line:
 } else if (profile == ALERT_PROFILE_VIBRATE_ONLY ) {
the else is not needed.
Also I believe that the 3 lines:
       NotificationManager mNotifyMgr = (NotificationManager) ctx.getSystemService(Context.NOTIFICATION_SERVICE);
        mNotifyMgr.cancel(Notifications.exportAlertNotificationId);
        mNotifyMgr.notify(Notifications.exportAlertNotificationId, builder.build());

should always happen, so there is no need to have them inside an if.